addons/mail/static/src/discuss/core/web/discuss_core_web_service.js

```diff
@@ -104,6 +104,15 @@ export class DiscussCoreWeb {
                 }
             }
         });
+        this.busService.subscribe("discuss.channel.member/unmute", (payload) => {
+            if (payload.persona.id === this.store.user.id) {
+                this.store.Thread.insert({
+                    id: payload.channel.id,
+                    model: "discuss.channel",
+                    notification_mute_until_dt: false,
+                });
+            }
+        });
```

Comment:
This is a notif to a single user, use `sendmany` on partner rather than channel, so it's sent only to that user.